source/material_model/visco_plastic.cc

```diff
@@ -362,11 +373,39 @@ namespace aspect
                              "instead of calculating the values through the specified thermal diffusivities, "
                              "densities, and heat capacities. ");
           prm.declare_entry ("Thermal conductivities", "3.0",
-                             Patterns::List(Patterns::Double(0)),
+                             Patterns::List(Patterns::Double (0.)),
                              "List of thermal conductivities, for background material and compositional fields, "
                              "for a total of N+1 values, where N is the number of compositional fields. "
                              "If only one value is given, then all use the same value. "
                              "Units: \\si{\\watt\\per\\meter\\per\\kelvin}.");
+          prm.declare_entry ("Use hydrothermal conductivity", "false",
+                             Patterns::Bool (),
+                             "Whether hydrothermal circulation is included in the calculation of the "
+                             "conductivity or not.");
+          prm.declare_entry ("Cutoff maximum temperature for hydrothermal conductivity", "873.15",
+                             Patterns::Double (0.),
+                             "Single value, declares up to which temperature hydrothermal conductivity is relevant. "
+                             "Units: \\si{\\kelvin}.");
+          prm.declare_entry ("Cutoff maximum depth for hydrothermal conductivity", "6000.0",
+                             Patterns::Double (0.),
+                             "Single value, declares up to which depth hydrothermal conductivity is relevant. "
+                             "Units: \\si{\\m}.");
+          //Think about that, Nusselt number should be >1 (so that conductivity is increased,
+          //is this correctly implemented?), but what default value does make sense? In paper is written "1-8"
+          prm.declare_entry ("Nusselt number for hydrothermal conductivity", "2.0",
+                             Patterns::Double (1.),
+                             "Single value, describes the Nusselt number, that is the ratio of convective "
+                             "to conductive heat transfer across the boundary of the crust. "
```

Comment:
Maybe add that increasing it will mimic the effect of faster circulating fluids and thus increase the thermal conductivity.